docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -56,6 +56,11 @@ On the **Dashboards** page, you can browse and manage folders and dashboards. Th
 As of Grafana 10.2, there is no longer a special **General** folder. Dashboards without a folder are now shown at the top level alongside folders.
 {{% /admonition %}}
 
+### Shared with me
+
+The **Shared with me** section displays folders and dashboards that are directly shared with a user which would otherwise not be visible in the standard dashboards tree.
+This is usually because the user doesn't have access to one or more of the parent folders.
```

Comment:
```suggestion
This is usually because you don't have access to one or more of the parent folders.
```

Reply:
Is there another use case that we should be mentioning? If it's not a relevant use case, can we just say, "This is because" and take out the word "usually"?